sbin/Release.sh

```diff
@@ -14,44 +14,48 @@
 #
 
 npm install
-# Expect a naming convention for build artifacts to follow this pattern:
-# OpenJDK<version>_<arch>_<os>_<timestampOrTag>.<extension>
-# Examples: OpenJDK8_x64_Windows_201813060547.zip, 
-#           OpenJDK8_x64_LinuxLH_201813060547.tar.gz, 
-#           OpenJDK10_aarch64_Linux_201813060547.tar.gz.sha256.txt,
-#           OpenJDKamber_x64_Linux_201813061304.tar.gz	
+
+timestampRegex="[[:digit:]]{4}-[[:digit:]]{2}-[[:digit:]]{2}-[[:digit:]]{2}-[[:digit:]]{2}"
+regex="OpenJDK([[:digit:]]+)U?_([[:alnum:]]+)_([[:alnum:]]+)_([[:alnum:]]+)_($timestampRegex).(tar.gz|zip)";
+regexArchivesOnly="${regex}$";
+regexAllFiles="${regex}(.sha256.txt)?";
+
+TIMESTAMP="$(date -u +'%Y-%m-%d-%H-%M')"
+
+# Rename to ensure a consistent timestamp across release
 for file in OpenJDK*
 do
-#                            1)ARCH         2)OS           3)TS_TAG       4)EXTENSION  5) SHA_EXT 
-  regex="OpenJDK[a-zA-Z0-9]+_([a-zA-Z0-9]+)_([a-zA-Z0-9]+)_([a-zA-Z0-9]+).(tar.gz|zip)(.sha256.txt)?";
   echo "Processing $file";
-  if [[ $file =~ $regex ]]; 
-  then 
-    ARCH=${BASH_REMATCH[1]};
-    OS=${BASH_REMATCH[2]};
-    TS_TAG=${BASH_REMATCH[3]};
-    EXTENSION=${BASH_REMATCH[4]};
-    SHA_EXT=${BASH_REMATCH[5]};
-    echo "version:${VERSION} arch:${ARCH} os:${OS} timestampOrTag:${TS_TAG} extension:${EXTENSION} sha_ext:${SHA_EXT}"; 
-  fi
-  if [ "$EXTENSION" == "zip" ]; 
-  then
-    FILENAME=`cat $file | awk  '{print $2}'`
-    sed -i -e "s/${FILENAME}/Open${VERSION}_${ARCH}_${OS}_${TS_TAG}.${EXTENSION}/g" $file
-  fi
-  if [ "$SHA_EXT" == ".sha256.txt" ]; 
+
+  if [[ $file =~ $regexArchivesOnly ]];
   then
-    mv $file "Open${VERSION}_${ARCH}_${OS}_${TS_TAG}${SHA_EXT}"
-  else
-    mv $file "Open${VERSION}_${ARCH}_${OS}_${TS_TAG}.${EXTENSION}"
+    newName=$(echo "${file}" | sed -r "s/${timestampRegex}/$TIMESTAMP/")
+
+    # Rename archive and checksum file with now timestamp
+    mv "${file}" "${newName}"
```

Comment:
I think a `echo Renaming ${file} to ${newName}` would be good in here, just to make it clear what's going on if anyone's trying to debug